content/en/guides/career-development.md

```diff
@@ -0,0 +1,224 @@
+Bryant said she and her team set out to create a career ladder that reflected those elements. They refitted the engineering career ladder to reflect similar skills for community managers. 
+
+*"Basically, there's a competency matrix that shows where someone's skill level should be in leadership, in collaboration, in community contribution, in technical acumen. And then, along that matrix, it will describe what the person's skill level should be at. So we took that very complex competency matrix and modified it to reflect what we thought some of the skills would be for community managers or what we call community architects,"* Bryant said.
+
+It took considerable and widespread collaboration to shape the career ladder from the beginning and finalize it as a fair and useful hiring and promotion tool in the end.
+
+*"It went through peer review and a pretty deliberate process to make sure that not only did it create a career path and an ability to celebrate someone's accomplishments in community architecture, but it also corresponded with an opportunity to increase their compensation in that model within the company,"* Bryant said.
+
+Once the career ladder was completed for the OSPO team and people were promoted accordingly, other departments with open source talent showed interest in it too.
+
+*"That's been really gratifying to see that get adopted by other teams in other areas of the company,"* Bryant said.
+
+True to open source roots and Red Hat’s culture, the company’s career guide is a work in progress, influenced by many voices working together towards an ever-changing definition of a useful tool. But whether you are just starting to develop an OSPO career guide or are working on one of many iterations, Bryant advises starting with the same focal point.
+
+*"Start with a good understanding of what you really need from your open source program team. It may not be the same for every company. Some companies work mostly on a regulatory side in terms of license compliance and some other work. My team doesn't work on that. For me, there's a certain core set of skills that are important to me, as there are in other offices,"* Bryant says. 
+
+## Integrating the unique with the standard
+
+Organizational variances are common even though patterns in how most companies think about and manage open source are similar. For example, all companies must decide how to handle compliance with open source software distribution, code contributions and code releases. There are also security, talent recruitment, developer relations, and other functions that companies must assign to a team to ensure all goes well. 
+
+"There's all these different things that every company deals with. But the subset of those things that happen to fall within the open source office is what's different at every company," said Norris. Google’s OSPO headed by Will Norris is an example of a team focused on engineering and compliance. A separate team handles the program side at Google. 
```

Comment:
In line 46 Will Norris is introduced as open source lead at Twitter. And here the text reads as if he's simultaneously heading Google's OSPO which may be confusing...

Reply:
Right! I've made a commit to fix this. Thank you for taking the time to review this! @choeppler